hal/src/argon/network/network.cpp

```diff
@@ -129,7 +129,11 @@ int if_init_platform(void*) {
     }
 
     if (HAL_Feature_Get(FEATURE_ETHERNET_DETECTION)) {
+#if PLATFORM_ID == PLATFORM_ARGON
         en2 = new WizNetif(HAL_SPI_INTERFACE1, D5, D3, D4, mac);
+#else // Argon SoM
+        en2 = new WizNetif(HAL_SPI_INTERFACE1, D8, 0xFF, D22, mac);
```

Comment:
We are setting `RESET` pin to `PIN_INVALID` here for some reason. Did we want to have `CS` pin to be set to the default instead?
```c++
WizNetif(HAL_SPI_Interface spi, pin_t cs, pin_t reset, pin_t interrupt, const uint8_t mac[6]);
```

Reply:
We are going to connect A7 to the Ethernet reset pin on the final mesh EVB. Will change it here in advance. It's not very meaningful to set the `cs` pin to a default, cos `D5` is not the default `CS` pin for mesh dev kits. Also for future platforms, it might not use the default `CS` pin for the Ethernet.